ipc/mdap.py

```diff
@@ -26,36 +26,66 @@ def connect_to_telemetry_server(address: tuple) -> socket.socket:
     print("Connection with the telemetry server established successfully.")
     return client_socket
 
+
 def telem_to_message(data_to_send, payloads):
     for telemetry in payloads[TelemetryPayload]:
         telemetry = cast(TelemetryPayload, telemetry)
-        data_to_send[f"{telemetry.module.name}-{telemetry.channel.name}"] = {
+
+        message = {
             "value": telemetry.data,
             "timestamp": telemetry.timestamp,
             "bucket": "mission_data"
         }
 
-        if (telemetry.module.name, telemetry.channel.name) == ('WatchdogHeartbeatTvac', 'AdcTempKelvin'):
-            app.logger.notice(f"BATTERY TEMP IS: {telemetry.data - 273.15}ºC")
-            data_to_send["TempC"] = {
-                "value": telemetry.data - 273.15,
-                "timestamp": telemetry.timestamp,
-                "bucket": "mission_data"
+        key = f"{telemetry.module.name}-{telemetry.channel.name}"
```

Comment:
this might be making the logic in the data ingestion server unnessarily complicated. Instead of the message being:

`{'telem1': [{}, ...]}`

maybe we can do:

{'all_telem': [{'name': 'telem1'}, {'name': 'telem1'}]}

Reply:
If I understand it correctly, we won't have `{telemetry.module.name}-{telemetry.channel.name}` as the dictionary's key anymore. Instead, we will add this in the message as a key-value pair (`'name': '{telemetry.module.name}-{telemetry.channel.name}'`) and have a list of all telemetry received. 

I believe having unique keys as `{telemetry.module.name}-{telemetry.channel.name}` will help in mapping them with the corresponding indicators in the future. Thoughts?